src/main/java/racingcar/InputView.java

```diff
@@ -0,0 +1,19 @@
+package racingcar;
+
+import java.util.Scanner;
+
+import static racingcar.RacingCar.toInts;
+
+public class InputView {
+    public static int numberOfCar(){
+        System.out.println("자동차 대수는 몇 대 인가요?\n");
+        return toInts((new Scanner(System.in)).nextLine());
+
+    }
+
+    public static int numberOfRound(){
+        System.out.println("시도할 회수는 몇 회 인가요?\n");
+        return toInts((new Scanner(System.in)).nextLine());
+    }
+
+}
```

Comment:
코드의 마지막은 마지막은 개행으로 끝나야 합니다 🙏 
```suggestion
}

```